src/foam/nanos/auth/Credentials.js

```diff
@@ -0,0 +1,25 @@
+foam.CLASS({
```

Comment:
Add header here and elsewhere, or leave it and we can do it.
/**
 * @license
 * Copyright 2024 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */